lib/src/components/FileDrop/utils.ts

```diff
@@ -8,7 +8,8 @@ export const isAnImage = (file: FileDropChildren['file']): boolean => {
   if (!file) return false
 
   if (typeof file === 'string') {
-    return !!getFileName(file).match(match)
+    const fileName = getFileName(file)
+    return !!fileName && !!fileName.match(match)
```

Comment:
why do we have that kind of changes ? it is less readable and does not do the same thing as before 🤔